### PR TITLE
Relace uses of MSG_LIMITS_STATUS  with MSG_FENCE_STATUS

### PR DESCRIPTION
**Pull request by peterbarker**
Only sub used this - and it sent a fence status message...

### CODE REVIEW

**Comment by rmackay9** [comment]
LGTM

File: ArduSub/GCS_Mavlink.cpp [hunk 1]
```diff
@@ -491,10 +491,10 @@ bool GCS_MAVLINK_Sub::try_send_message(enum ap_message id)
 #endif
         break;
 
-    case MSG_LIMITS_STATUS:
+    case MSG_FENCE_STATUS:
 #if AC_FENCE == ENABLED
-        CHECK_PAYLOAD_SIZE(LIMITS_STATUS);
-        sub.send_limits_status(chan);
```
Comment (OXINARF) [inline ArduSub/GCS_Mavlink.cpp]: If the method is not used (which I agree with), then remove it please.

**Comment by rmackay9** [comment]
Merged!

.. although i probably should have waited for @jaxxzer's feedback..

**Comment by jaxxzer** [comment]
fine, ty.

**Pull request merged**